.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,16 @@
+name: auto-merge
+
+on:
+  pull_request_target:
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+    - uses: ahmadnassri/action-dependabot-auto-merge@v2.6
+      with:
+        github-token: '${{ secrets.ROBOT_ROX_GITHUB_TOKEN }}'
```

Comment:
@SimonBaeumer since I have no rights to this repo could you add this secret?

Reply:
Added you @janisz (and Draco and Automation) as Admins.

Reply:
I've added secret